docs/develop/guides/working-with-momento-tokens.md

```diff
@@ -0,0 +1,131 @@
+3. Valid Until: this is purely informational, and displays the date and time the current auth token expires at.
+
+![image](/img/getting-started/generated-token.jpg)
+
+## Step 3: Store your Momento Token
+If you wish to quickly test out Momento, click on the copy icon beside the auth token to copy the token to your clipboard and supply it to the Momento SDK. We recommend you also click the "Download JSON" button to store both the auth token and refresh token. Tokens are like passwords, the best practice is to store it in a secure location such as AWS Secrets Manager or GCP Secrets Manager.
+
+Here is an example of using the auth token in the Momento Javascript SDK, and then refreshing it to obtain a new token before it expires.
+<Tabs>
+    <TabItem value="nodejs" label="Node.js" default>
+
+```javascript
+const fs = require('fs');
+const { AuthClient, GenerateAuthToken, RefreshAuthToken, ExpiresAt, ExpiresIn } = require('@gomomento/sdk');
+const dotenv = require('dotenv');
+
+// Value must contain a valid go momento endpoint
+const controlEndpoint = "valid-gomomento-endpoint";
+// Value must contain a valid momento auth token
+const authToken = "valid-momento-auth-token";
+
+dotenv.config();
+
+// Creates the Momento auth client object
+async function createAuthClient() {
+  return new AuthClient();
+}
+
+// Creates the Momento cache client object
+async function createCacheClient(momentoAuthToken: string) {
+  return new CacheClient({
+    configuration: Configurations.Laptop.v1(),
+    credentialProvider: CredentialProvider.fromString({
+      environmentVariableName: momentoAuthToken,
+    }),
+    defaultTtlSeconds: 600,
+  });
+}
+
+async function refreshExpiringToken(authToken: string, refreshToken: string) {
+  const refreshTokenResponse = await authClient.refreshAuthToken(
+    CredentialProvider.fromString({
+      authToken: authToken,
+    }),
+    refreshToken
+  );
+
+  if (refreshTokenResponse instanceof RefreshAuthToken.Error) {
+    throw new Error(`Error: ${refreshTokenResponse.message()}`);
+  }
+  console.log('Auth token successfully refreshed!');
+  
+  return refreshTokenResponse;
+}
+
+async function usingAuthToken(cacheClient: CacheClient) {
+  const result = await cacheClient.listCaches();
+  if (result instanceof ListCaches.Success) {
+    console.log(
+      `Caches:\n${result
+        .getCaches()
+        .map(c => c.getName())
+        .join('\n')}\n\n`
+    );
+  } else if (result instanceof ListCaches.Error) {
+    throw new Error(
+      `An error occurred while attempting to list caches: ${result.errorCode()}: ${result.toString()}`
+    );
+  }
+}
+
+async function run() {
+  const byteArray = await readFile(filePath);
+  if (byteArray === null) {
+    return;
+  }
+
+  const authClient = await createAuthClient();
+
+  const cacheClient = await createCacheClient(authToken);
+  
+  await usingAuthToken(cacheClient);
+  
+  const refreshTokenResponse = await refreshExpiringToken(authToken, refreshToken);
```

Comment:
@ben-kugler @rlinehan @cprice404 this code example is too long, any ideas on how to shrink it way down to say, 10-20 lines? I don't expect users to use this code in production, just want to illustrate literally how to use auth token to call Momento API (e.g. listCaches), and how to refresh an auth token. Basically, L116-L118 is the meat of what I want to show.

Reply:
1. I don't see where this readFile stuff is being used so there are a bunch of lines there that look like they could be removed
2. we could remove all of the error handling branches if you want.  i'm dubious about whether that's a good idea... if people come to this page looking for information on security best practices i'm not sure i believe they don't expect to use the code in production
3. This code probably won't even compile any more against the current HEAD because the AuthClient API is not stable yet; this is why I don't like having code snippets in docs that don't get tested in CI.  I've got a pattern for pulling in CI-tested code snippets into the docs from the SDK repos now, and I am planning to circle back and patch a bunch of places in the docs where we are directly embedding stuff like this and migrate it over to that pattern.  I expected to do that this week but I haven't had time because other things related to this API have come up.

my 2c on this particular code snippet:

1. can we just remove it for now and say `use the AuthClient`, just to hold us over for the next few days
2. I would love to know how others feel about verbosity vs. production-readiness here, I want to do the right thing for our users
3. As soon as I am freed up from the remaining AuthClient/authz work I will come work on updating all the in-lined node.js code examples to pull from the repo instead, and I will add back in some working, tested example code for this

lmk what you think